cmd/deploy/deploy.go

```diff
@@ -272,6 +272,10 @@ func getProxyHandler(name, token string, clusterConfig *rest.Config) (http.Handl
 	proxy := httputil.NewSingleHostReverseProxy(destinationURL)
 	proxy.Transport = trans
 
+	isOktetoRepo, err := utils.IsOktetoRepo()
```

Comment:
based on comment below on usage of these returned vars you might change this.
```suggestion
        var isOktetoRepo bool
	is, err := utils.IsOktetoRepo()
	if err == nil {isOktetoRepo = is} else{isOktetoRepo=false; log.info(....)}
```